src/common/viz/CustomConfigWidgets.json

```diff
@@ -0,0 +1,7 @@
+[
```

Comment:
In terms of keeping this logic generic so it can more easily become a standalone library, it would be better to implement using component settings. 

This would allow the configuration for it to be specified [here](https://github.com/deepforge-dev/deepforge/blob/master/config/components.json) and projects could define custom widgets - much like we are doing in this PR.